people/syphar.toml

```
@@ -0,0 +1,4 @@
+name = 'Denis Cornehl'
+github = 'syphar'
+github-id = '540890'
+email = 'denis.cornehl@gmail.com'
```

```suggestion
email = 'denis@cornehl.org'
```

(sorry, just in the middle of a migration process)